next/components/ui/NavBarHeader/NavBarHeader.tsx

```diff
@@ -0,0 +1,112 @@
+import SearchIcon from '@assets/images/search-icon.svg'
+import { isItExternal } from '@bratislava/ui-bratislava/BAStickyMenu/external-link'
+import { useGeneralContext } from '@utils/generalContext'
+import { isDefined } from '@utils/isDefined'
+import SkipToContentButton from 'components/molecules/SkipToContentButton'
+import CookieConsent from 'components/organisms/CookieConsent'
+import { useTranslation } from 'next-i18next'
+import React from 'react'
+
+import Button from '../../forms/simple-components/Button'
+import MLink from '../../forms/simple-components/MLink'
+import { Brand } from '../Brand/Brand'
+
+type NavBarProps = { className?: string } & LanguageSelectProps
+
+const Divider = ({ className }: { className?: string }) => {
+  return <div aria-hidden className={`border-r h-6 ${className}`} />
+}
+
+export const NavBarHeader = ({ className, ...languageSelectProps }: NavBarProps) => {
+  const { t, i18n } = useTranslation(['common'])
+
+  const { general } = useGeneralContext()
+  const { header } = general?.data?.attributes ?? {}
+  const { links, accountLink } = header ?? {}
+
+  const otherLanguage = languageSelectProps.languages?.find((l) => l.key !== i18n.language)
+
+  return (
+    <>
+      <div className="fixed top-0 left-0 w-full bg-white z-40">
+        <div className="flex max-w-screen-lg m-auto h-[57px] w-full items-center justify-between border-b border-gray-200">
+          <SkipToContentButton />
+          <Brand
+            className="group"
+            url="/"
+            title={
+              <div className="text-p2 text-font group-hover:text-gray-600">
+                {i18n.language === 'en' && <span className="font-semibold">Bratislava </span>}
```

Comment:
Pls.

https://github.com/bratislava/kupaliska-starz-fe/blob/c6ef9e0aa287e9fa5d9403cb0dc02240d1a8510a/src/components/HeroBanner/HeroBanner.tsx#L19
https://github.com/bratislava/kupaliska-starz-fe/blob/c6ef9e0aa287e9fa5d9403cb0dc02240d1a8510a/public/locales/sk/translation.json#L204

Reply:
https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0

Reply:
Sorry, it wasn't in the scope of this PR, but noted :)